{
  "debianbug": 719118,
  "description": "The (1) backup (api/contrib/backups.py) and (2) volume transfer (contrib/volume_transfer.py) APIs in OpenStack Cinder Grizzly 2013.1.3 and earlier allows remote attackers to cause a denial of service (resource consumption and crash) via an XML Entity Expansion (XEE) attack.  NOTE: this issue is due to an incomplete fix for CVE-2013-1664.",
  "releases": {
    "bullseye": {
      "fixed_version": "2013.1.2-4",
      "repositories": {
        "bullseye": "2:17.0.0-2"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "2013.1.2-4",
      "repositories": {
        "buster": "2:13.0.3-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "2013.1.2-4",
      "repositories": {
        "sid": "2:17.0.0-2"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "2013.1.2-4",
      "repositories": {
        "stretch": "2:9.0.0-4"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}